src/Refined/Unsafe.hs

```diff
@@ -28,6 +28,7 @@
 --------------------------------------------------------------------------------
 
 {-# LANGUAGE CPP #-}
+{-# LANGUAGE FlexibleContexts #-}
```

Comment:
why is this needed now?

Reply:
Due to this https://gitlab.haskell.org/ghc/ghc/-/wikis/migration/9.0#more-validity-checking-in-quantified-constraints